include/FockSpace/BaseFockSpace.hpp

```diff
@@ -36,6 +37,17 @@ class BaseFockSpace {
     virtual ~BaseFockSpace() = 0;
 
 
+    // GETTERS
+    size_t get_dimension() const { return dim; }
+    size_t get_K() const { return K; }
+
+
+    // PUBLIC METHODS
+    /**
+     *  Creates a Hartree-Fock guess
+     */
+    Eigen::VectorXd hartreeFockGuess();
```

Comment:
I don't think 'guess' is a good name here. Maybe just `HartreeFockExpansion()` or so?